README.md

```diff
@@ -26,7 +38,213 @@ required for testing citus.
 * [Advanced fab Usage](#advanced-fab)
   * [Using Multiple Citus Installations](#multiple-installs)
 
-# <a name="getting-started"></a> Getting Started
+## <a name="azure"></a>Azure
+
+## <a name="azure-getting-started"></a> Getting Started
+
+You can find more information about every step below in other categories. This list of commands show how to get started quickly. Please see other items below to understand details and solve any problems you face.
+
+You should have `az cli` in your local to continue.
+
+You should use `ssh-agent` to add your ssh keys, which will be used for downloading the enterprise repository. Note that your keys are kept only in memory, therefore this is a secure step.
+
+In `azuredeploy.parameters.json` file, you will see the parameters that you can change. For example if you want to change the number of instances, you will need to change the parameter `numberOfInstances`. You can change the type of coordinator and workers separately from the parameters file. Also by default for workers, memory intense vms are used(E type) while for coordinator CPU intense vms are used(D type).
```

Comment:
Minor: `azuredeploy.parameters.json` -> `azure/azuredeploy.parameters.json`

or probably add one step below `cd azure` and all other things also would work fine

Reply:
I think it shouldnt matter where we run this script from so I will cd into the 'azure' folder from wherever the script is run with:

```bash
azuredir="${0%/*}"
cd ${azuredir}
```

Now we can run the script from any folder.

Reply:
It seems added currently.